components/AdForm/__tests__/__snapshots__/formats.tests.js.snap

```diff
@@ -12,11 +12,17 @@ Object {
             "items": Array [
               "post",
               "page",
-              "media",
             ],
             "position": 1,
             "type": "single",
           },
+          Object {
+            "items": Array [
+              "media",
+            ],
+            "position": 1,
```

Comment:
I'd change these `"position"` from numbers to strings so they are consistent with the new implementation:
https://github.com/frontity/ads-form/blob/33ce4532a4b02297dc48d7dc757608b147be2e3d/components/AdForm/__tests__/formats.tests.js#L27-L48